### DISCUSSION TITLE
CKAN monthly meetups

### DISCUSSION THREAD

**Discussion by popovayoana** [ckan/ckan#6065] [Show and tell]
## Description
The purpose of this discussion is to give a clear and concise structure to the process of organizing CKAN monthly meetups. By following these steps anybody should be able to organize a meetup. _Note: Any suggestions and comments are much appreciated._

# CKAN Monthly Live
CKAN Monthly Live is a series of monthly meetups to showcase CKAN to people. It happens every third Wednesday of the month at 13:00 UTC, it’s free to join and open to everyone. Each event includes a single session and a live Q&A forum. It’s an opportunity to connect with the speakers, the other attendees and the whole CKAN community.

**Frequency:** Every third Wednesday of the month at 13:00 UTC (https://www.google.com/search?q=1pm+utc)

## Agenda

-  Link to [meeting doc](https://docs.google.com/document/d/19eTfNASlPSMU4E5KeTTB6cLyK6_XCfD-zy7AjqFVgS8/edit?usp=sharing)

## Acceptance

- [ ] Announcements on social media
- [ ] Email sent to everybody from RSVP list
- [ ] Blog post for the meetup (recap)
- [ ] Recording uploaded to YouTube
- [ ] Material prepped re how people can contribute
- [ ] An awesome 1h community meetup with at least 15 participants
- [ ] Follow up email sent to everybody (survey)

## Tasks

- [ ]  Generate a Zoom call link
- [ ]  Reserve slot in Zoom account
- [ ] Announce CKAN Monthly Live on social media (LinkedIn, Twitter, Facebook)
- [ ] Send email to RSVP list and people from ckan.org webinar form
- [ ]  Send a reminder to register a week before the meetup
- [ ]  Send a reminder to register 3 days before the meetup
- [ ]  Email guests ~2-3h before the event with the meeting details
- [ ]  Announce on Twitter
- [ ]  Send reminders on Gitter
- [ ]  Announce in GChat (if relevant) a few hours before the meetup
- [ ]  Send meeting link ~10 minutes before the event on Gitter
- [ ]  Record the meeting
- [ ]  Keep meeting minutes in [CKAN Meetups Meeting Doc [Public]](https://docs.google.com/document/d/19eTfNASlPSMU4E5KeTTB6cLyK6_XCfD-zy7AjqFVgS8/edit?usp=sharing)
- [ ]  Material prepped re how people can contribute
- [ ]  Schedule next meeting and update the RSVP form with the correct meetup date → https://forms.gle/kE5TbH5YhF3EVhRz8
- [ ]  Announce the date of the next meeting (through new GitHub issue and Gitter)
- [ ]  Add previous guests to the Google Calendar invite (make sure Google Meet is turned off!)
- [ ]  Notify guests by email about the calendar event
- [ ]  Create a new entry in the public [meeting doc](https://docs.google.com/document/d/19eTfNASlPSMU4E5KeTTB6cLyK6_XCfD-zy7AjqFVgS8/edit?usp=sharing) with agenda for this meetup
- [ ]  Quick recap on social media, Gitter, etc 
- [ ]  Upload the video - process video(s) (cut presentation out as a separate video if any, trim beginning and end if necessary)
- [ ]  Write a blog post on ckan.org and when recording is done, update blog post with link to YouTube video
- [ ] Send survey to people

## Suggested topics

- The role of Data Portals in the Third Wave (publication - Third Wave of Open Data by GovLab) (speakers: Steven or sb else)
- Data Portals in Enterprise (Enterprise Data Management with CKAN) (speakers: Steven or sb else)
- Mission, Systems and Transparency - the many faces of a data portal (speakers: Steven or sb else)
- AI and Ethics (speakers: Steven or sb else)
- Civic Innovation - (case study) (speakers: Steven or sb else)
- Behind the scenes look at Data Portals 
  - SEED Data Portal (speakers: Steven or sb else)
  - Floodplains (speakers: Steven or sb else)
  - Pacificdata.org (speakers: Steven or Sioeli Tonga)
- Aids Data Repository demo and overview (speaker: Rufus or somebody else)
- WPRDC update … (speaker: Rufus or somebody else)
- Overview or deep dives of popular extensions
- _Add topic and suggest a speaker here: [Suggestions for Topics and Speakers](https://docs.google.com/document/d/1HnNi-3kLLVWU0Daf53uLdZDdrgSV1yWVMfrIWb8_nGM/edit?usp=sharing)_

## Documents

- [CKAN Meetups Meeting Doc [Public]](https://docs.google.com/document/d/19eTfNASlPSMU4E5KeTTB6cLyK6_XCfD-zy7AjqFVgS8/edit?usp=sharing) - this is the document where we (a moderator) will keep our meeting notes from each meetup
- [Google RSVP form](https://forms.gle/kE5TbH5YhF3EVhRz8)
- [Running CKAN Community Meetups](https://docs.google.com/document/d/1CRqaokZLPEz6HK8lGT5vbFRmvju5stOj2Qpahya3rpg/edit?usp=sharing) - a more detailed working document on how community meetups will be organized; _note: you can edit, comment, make suggestions, add information (anything u find useful is very welcomed)_

**Comment by anuveyatsu** [comment]
@popovayoana I'm asking some of our clients if they want to present their data portals and talk about their use case in one of these meetings.

**Reply by anuveyatsu** [reply]
@popovayoana how do I propose speakers?

**Reply by popovayoana** [reply]
@anuveyatsu added a file - [Suggestions for Topics and Speakers](https://docs.google.com/document/d/1HnNi-3kLLVWU0Daf53uLdZDdrgSV1yWVMfrIWb8_nGM/edit?usp=sharing)

**Reply by anuveyatsu** [reply]
@popovayoana when are we planning to have the meetup now? Original date was 17th May 😄

**Reply by popovayoana** [reply]
There's a new date suggested - 2nd of June, 3 pm CET. We have already set up the first speaker - Steven De Costa. Need to ask if somebody is willing to be a moderator tho. Will update and announce very soon. Meanwhile, if you have some useful thoughts / ideas, or are willing to moderate the first webinar, let me know : ))

**Reply by anuveyatsu** [reply]
@popovayoana just to confirm - you mean 3 pm CET as 2 pm GMT or 1 pm GMT?

**Reply by popovayoana** [reply]
@anuveyatsu 1 pm GMT. it's based on the availability of Steven and the assumption that this will be okay for most people. wdyt?

**Comment by jqnatividad** [comment]
This is a much needed and welcome initiative!

Would it be possible to present and demo the revamped DataTables view and the time series Datapump?

Is there a sign-up sheet where I can submit an abstract/proposal?

Thanks!
Joel

**Reply by popovayoana** [reply]
@jqnatividad Awesome! I think we should have a "single source of truth" so for now you could just write your suggestion here: [Suggestions for Topics and Speakers](https://docs.google.com/document/d/1HnNi-3kLLVWU0Daf53uLdZDdrgSV1yWVMfrIWb8_nGM/edit?usp=sharing). And by the way, if u have a suggestion for a smarter way to propose topics, feel free to share it. Any refinement ideas are super welcomed. We wanna smooth things over, so that everything flows well ; )

**Comment by popovayoana** [comment]
@wardi I had to drop quickly and I am not sure if I did everything properly so I will duplicate my thoughts here. So, about what you were saying... I will be very glad to discuss the topic and I am ready to help / cooperate with whatever needed. My email is **yoana.popova@datopian.com**. Feel free to share it. ; ) 

P.S. That's probably not the best way to contact you, haha, but... I hope it's okay. ; )
